### PR TITLE
docs: 更新README文件以完善项目信息

### PR DESCRIPTION
**Pull request by eight14**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/xun082/create-neat/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation update
- [ ] Performance improvement
- [ ] Test related changes
- [ ] Security fix
- [ ] Dependency update
- [ ] Revert changes
- [ ] Other... Please describe:

## What is the current behavior?

- Closes #
- Related to #

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,223 +1,86 @@
...
-- 🥂 使用 `Webpack` 打包你的 `Web` 应用程序,实现多环境打包部署，代码分割优化，配合官方分析工具，实时优化代码;
-- 🍻 支持用户自定义 `Webpack` 配置扩展原有的配置，为项目添加特有的功能;
-- 🎯 支用户自定义 `Babel` 配置，让你的程序更健壮;
-- 📕 友好的日志输出，让你快速定位问题所在以及增加开发体验;
-- 在使用 `Husky` 的情况下使用 `standard-version` 自动生成 `CHANGELOG` 文件;
-- 🔸 等等......
+- 📦 **零配置，开箱即用。**
+- 🚀 **插件式项目生成**：根据用户需求灵活生成不同基础项目。
+- 🧠 **支持无限扩展的架构**：框架、构建工具和插件的潜力无限。
+- 💯 **丰富的使用教程和代码注释**：帮助每位社区成员快速学习前端工程化相关知识。
```
Comment (uaenaTzx) [inline README.md:25]: 要不额外开一个板块，介绍我们的社区，把这两个放进去。
因为这个不能算是这个 cn 的技术特点

File: README.md [hunk 2]
```diff
@@ -1,223 +1,86 @@
...
-    return config;
-  },
-};
-```
-
-### 五、贡献 🍵
+### 六、贡献指南
 
-> 如果你正在使用这个项目或者对这个项目感兴趣，可以通过以下方式支持我:
+[查看贡献指南文档]()
```
Comment (uaenaTzx) [inline README.md:78]: 链接没有

File: README.md [hunk 3]
```diff
@@ -1,223 +1,86 @@
...
-#### Example
-
-```js
-// babel.config.js
-
-module.exports = {
-  plugins: ["babel-plugin-styled-components"],
-};
-```
+![项目创建成功](https://p3-juejin.byteimg.com/tos-cn-i-k3u1fbpfcp/fe46e77448d54c14aebf4557db949883~tplv-k3u1fbpfcp-zoom-1.image)
```
Comment (uaenaTzx) [inline README.md:58]: 图片上有掘金水印

File: README.md (hunk 2 above)
Comment (eight14) [reply README.md]: 已加上

File: README.md (hunk 1 above)
Comment (eight14) [reply README.md:26]: 嗯嗯好

**Review by uaenaTzx — APPROVED** [review]

File: README.md
Comment (eight14) [inline README.md:1]: 先别审，格式有误

**Review by uaenaTzx — APPROVED** [review]

**Review by uaenaTzx — APPROVED** [review]

**Review by uaenaTzx — APPROVED** [review]

**Pull request merged**